angstudy/src/app/components/basket-num/basket-num.component.html

```diff
@@ -1,11 +1,9 @@
-<mat-icon matBadge={{basketNum}} matBadgeColor="warn"> 
-    <img 
-      loading="auto" 
-      src="assets/icons/icon-basket.svg"
-      alt="favorite" 
-    />
-  </mat-icon>
-  <!-- Include text description of the icon's meaning for screen-readers -->
-  <span class="cdk-visually-hidden">
-    Корзина
-  </span>
\ No newline at end of file
+<mat-icon 
+	matBadge="{{ basketNum }}" 
+	matBadgeColor="warn"
+	[routerLink]="'/basket'"
```

Comment:
```suggestion
	routerLink="/basket"
```
У тебя routerLink статичны, по этому биндинг для него избыточен)

Reply:
Исправил. Не знал что для статичных так надо

Reply:
Не то что бы надо, можно и через квадратные, но зачем, если можно разгузить шаблон + так сразу выделяются в шаблоне статичные и динамичные инпуты/атрибуты